lib/util/util.dart

```diff
@@ -50,4 +52,15 @@ class Util {
       return false;
     }
   }
+
+  static void showToast(BuildContext context, String text) {
```

Comment:
perhaps this should have its own file?

Reply:
also maybe the file name should be different util is not very descriptive

Reply:
I don't think a single method should have its own file